### ISSUE TITLE
RFID Raw - Keri Systems PKT-10X Standard Light Proximity Key Tag

### ISSUE THREAD

**Issue by JReming85** [flipperdevices/flipperzero-firmware#2777]
### Describe the enhancement you're suggesting.

Would love it if you could implement this RFID tag.

They are the key tag our area uses to allow us to get into the mail and laundry room and pool.

We only get 1, and if we lose it we have to pay $50 for a replacement.  Would rather just clone it and keep the original safe.
[rfid backups.zip](https://github.com/flipperdevices/flipperzero-firmware/files/11764822/rfid.backups.zip)

**Comment by skotopes** [comment]
Some keri tags are already supported.
Raw files looks a little bit strange, looks like we need PKT-10X at our hands to implement them.

Also do you have proxmark?

**Comment by JReming85** [comment]
Sorry I do not.  I can rescan raws of the tag if that will help

**Comment by JReming85** [comment]
Here are 2 additional raw scans I just did away from all my other equipment.  Hopefully it helps

[raw.zip](https://github.com/flipperdevices/flipperzero-firmware/files/11833593/raw.zip)

**Comment by skotopes** [comment]
Yep, it may be rfid, but coding is quite unusual. We'll try to implement it

**Comment by jbeoris** [comment]
Any status on this? @skotopes @doomwastaken ? I have the same issue.

**Comment by skotopes** [comment]
@jbeoris in backlog waiting for someone to implement it

**Comment by forxhunter** [comment]
please support this !!!

**Comment by Ravioli-cpu** [comment]
Interested as well

**Comment by Ravioli-cpu** [comment]
@skotopes @doomwastaken hello, was wondering if there were any updates? Thanks so much
